debezium-core/src/main/java/io/debezium/relational/history/DatabaseHistory.java

```diff
@@ -61,7 +61,9 @@
                                                 .withDefault(
                                                     "DROP TEMPORARY TABLE IF EXISTS .+ /\\* generated by server \\*/," +
                                                     // Filter out RDS heartbeat statements, see DBZ-469
-                                                    "INSERT INTO mysql.rds_heartbeat2\\(.*\\) values \\(.*\\) ON DUPLICATE KEY UPDATE value = .*"
+                                                    "INSERT INTO mysql.rds_heartbeat2\\(.*\\) values \\(.*\\) ON DUPLICATE KEY UPDATE value = .*" +
```

Comment:
The statements should be comma-separated. A test on CI caught it, that's good. Probably a good idea though to expand the test a bit :)